entity/index.js

```diff
@@ -765,6 +806,20 @@ EntityGenerator.prototype.files = function files() {
 
     var resourceDir = 'src/main/resources/';
 
+    for (var fieldIdx in this.fields) {
+        var field = this.fields[fieldIdx];
+        if (field.fieldIsEnum == true) {
+            var fieldType = field.fieldType;
+            var lastDot = fieldType.lastIndexOf(".");
+            var enumInfo = new Object();
+            enumInfo.packageName = fieldType.substring(0, lastDot);
+            enumInfo.enumName = fieldType.substring(lastDot + 1);
+            enumInfo.enumValues = field.fieldValues;
+            this.template('src/main/java/package/_Enum.java',
+                'src/main/java/' + fieldType.replace(/\./g, '/') + '.java', enumInfo, {});
```

Comment:
In fact, I request the user to enter a class name, but what I mean, is to enter a "full class name with package" (line 198). I will change the text to make things clearer if you agree?